app/src/main/res/layout/component_recent_bookmarks.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:isScrollContainer="true"
+    android:gravity="start">
+
```

Comment:
Shouldn't the contents of `recent_bookmarks_header` be here (above the `RecyclerView`)? Is there a situation where you can have a bookmarks header without the list?

You can use `<include layout="" />` if you want to keep some separation of concerns between the header and this instead of in-lining.

Reply:
Great point! I want to keep them separate for now in case we decide to do a placeholder component (like collections), so I think `<include layout..>` would be a good solution.